### DISCUSSION TITLE
Change IP?

### DISCUSSION THREAD

**Discussion by benjarrell** [logdyhq/logdy-core#12] [Q&A] [answered]
Is there way to change the IP it listens on?

**Comment by PeterOsinski** [comment]
Hi @benjarrell do you mean the IP that the UI is served on or the IP of socket mode?
[docs link](https://logdy.dev/docs/reference/cli#ip-used-with-forward-and-socket-mode-only)

**Reply by benjarrell** [reply]
The address the UI is served on. The use case is on headless servers, a single executable is a lot easier to pull off bureaucratically than also needing to having a full blown httpd server installed.

**Reply by PeterOsinski** [reply, marked as answer]
That's a valid use case, will include that in the next release, thanks @benjarrell

**Reply by PeterOsinski** [reply]
@benjarrell you can now set IP with `--ui-ip` setting, [read more in v0.7 announcement](https://logdy.dev/blog/post/logdy-new-version-announcement-v07)
